bee-test/src/rand/output.rs

```diff
@@ -82,12 +96,13 @@ pub fn rand_foundry_output() -> output::FoundryOutput {
 pub fn rand_nft_output() -> output::NftOutput {
     let feature_blocks = rand_allowed_feature_blocks(output::NftOutput::ALLOWED_FEATURE_BLOCKS);
     output::NftOutput::build(
-        rand_number(),
+        rand_number_range(Output::AMOUNT_RANGE),
         output::NftId::new(rand_bytes_array()),
         rand_bytes(rand_number_range(0..output::NftOutput::IMMUTABLE_METADATA_LENGTH_MAX) as usize),
     )
     .unwrap()
     .with_feature_blocks(feature_blocks)
+    .add_unlock_condition(AddressUnlockCondition::new(rand_alias_address().into()).into())
```

Comment:
- This doesn't have to be an alias address only
- This should be different from the nft_id (but can be an nft address)